.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+# Runs testing related jobs.
+
+name: test
+
+on:
+  workflow_call:
+    inputs:
+      target_branch:
+        description: 'The branch to deploy a book from'
+        required: false
+        type: string
+        default: 'next'
+
+# Limits workflow concurrency to only the latest commit in the PR.
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  test:
```

Comment:
`doc-test` would be good to run as well as part of the generic `test.yml`

Reply:
added